integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMultiDomainModelsScale.java

```diff
@@ -141,7 +112,6 @@ class ItMultiDomainModelsWithLoadBalancer {
   private static final String WLDF_OPENSESSION_APP = "opensessionapp";
   private static final String WLDF_OPENSESSION_APP_CONTEXT_ROOT = "opensession";
   private static final String wlSecretName = "weblogic-credentials";
-  private static final String DATA_HOME_OVERRIDE = "/u01/mydata";
```

Comment:
There is no rolling restart happening on the class as mentioned on class level javadoc  "rolling restart behavior in a multi-cluster MII domain "